libcxx/docs/Status/Cxx23Papers.csv

```diff
@@ -120,4 +120,4 @@
 "`P2614R2 <https://wg21.link/P2614R2>`__","LWG", "Deprecate ``numeric_limits::has_denorm``","February 2023","|Complete|","18.0",""
 "`P2588R3 <https://wg21.link/P2588R3>`__","LWG", "``barrier``’s phase completion guarantees","February 2023","","",""
 "`P2763R1 <https://wg21.link/P2763R1>`__","LWG", "``layout_stride`` static extents default constructor fix","February 2023","","",""
-"`P2736R2 <https://wg21.link/P2736R2>`__","CWG","Referencing The Unicode Standard","February 2023","","","|format|"
+"`P2736R2 <https://wg21.link/P2736R2>`__","CWG","Referencing The Unicode Standard","February 2023","Nothing to do","","|format|"
```

Comment:
Not sure whether this should be considered completed in #86543 (and hence should be in 19.0). @mordante

Reply:
Hmm, the referenced Unicode version used by `<format>` seemed to be Unicode 14? (per Bibliography in the paper)

Reply:
This paper, in my opinion, just removes the referenced version - the current C++ standard only refers to the latest unicode standard, unversioned. In my opinion, this means that implementations are free to reference any standard (perhaps only newer than the C++ standard they are implementing), and therefore that updating the standard referenced in implementations is just an implementation detail.

Reply:
Actually it means implementations should _always_ use the latest released Unicode version. This means it's complete until the next Unicode release. I think marking it as complete in version 19.0 would be better.

There is a [core issue](https://github.com/cplusplus/papers/issues/1736) filed against this change.

Reply:
Mmh, point; I'll make this change, thanks @mordante!